### ISSUE TITLE
Backup Options

### ISSUE THREAD

**Issue by timelady-victorious** [shedaniel/your-options-shall-be-respected#3]
To make it more different to DefaultSettings Fabric (and more useful imo) YOSBR should copy options.txt and config/ whenever you _close_ minecraft. This means if options.txt is updated by a modpack, you can go right back to your options instead of some options from ages ago. This should be configurable.

**Comment by shedaniel** [comment]
Not sure what you mean, the options file is only copied if the file doesn't exist.

**Comment by timelady-victorious** [comment]
Yeah, but recopy it every single time you close the game, or maybe with a configerable keypress would be cool.

**Comment by Madis0** [comment]
That would defy the point of the mod, which is to _respect_ the user's settings... #6 idea is better as the user would voluntarily do it.
